specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -114,34 +115,31 @@ namespace Microsoft.EventGrid {
     value: ReceiveDetails[];
   }
 
-  @doc("Failed lock token information.")
+  @doc("Failed LockToken information.")
   model FailedLockToken {
-    @doc("The lock token of an entry in the request.")
+    @doc("LockToken value")
```

Comment:
That just reiterates the property name. It doesn't describe what the value is. Maybe expand on this more, at least for GA (not blocking)?